src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Markup/Baml2006/Baml2006Reader.cs

```diff
@@ -2065,15 +2065,15 @@ private string Logic_GetFullyQualifiedNameForType(XamlType type)
 
         private string Logic_GetFullXmlns(string uriInput)
         {
-            int colonIdx = uriInput.IndexOf(":");
+            int colonIdx = uriInput.IndexOf(':');
             if (colonIdx != -1)
             {
                 string uriTypePrefix = uriInput.Substring(0, colonIdx);
                 if (String.Equals(uriTypePrefix, "clr-namespace"))
```

Comment:
This IndexOf+Substring+Equals is pretty inefficient, a simple StartsWith would be faster (optionally with a check for ':' at a const index(13) before to short-circuit).